client/i18n/config.js

```diff
@@ -12,20 +14,67 @@ i18n.use(initReactI18next).init({
   fallbackLng: 'en',
   lng: i18nextCode,
   // we only load one language since each language will have it's own server
+  // They need to be evaluated ahead of time, to prevent Webpack from bundling
+  // the entire locales directory.
   resources: {
     [i18nextCode]: {
-      translations: require(`./locales/${clientLocale}/translations.json`),
-      trending: require(`./locales/${clientLocale}/trending.json`),
-      intro: require(`./locales/${clientLocale}/intro.json`),
-      metaTags: require(`./locales/${clientLocale}/meta-tags.json`),
-      links: require(`./locales/${clientLocale}/links.json`)
+      translations: preval`
+      const envData = require('../../config/env.json');
+      const { clientLocale } = envData;
+      // to avoid double imports, when the locale is english
+      if (clientLocale === 'english') {
+        module.exports = {};
+      } else {
+        module.exports = require('./locales/' + clientLocale + '/translations.json');
+      }
+    `,
```

Comment:
A question - since we specifically export the english files below, what if we do nothing for english here?

```js
if (clientLocale !== 'english') {
  module.exports = require('./locales/'...)
  }
```

Reply:
Not sure, but it's a good idea.  I'll try it.